src/java.base/share/classes/javax/crypto/spec/HKDFParameterSpec.java

```diff
@@ -312,11 +317,11 @@ private Extract(List<SecretKey> ikms, List<SecretKey> salts) {
         }
 
         /**
-         * Returns an unmodifiable {@code List} of input key material values in
-         * the order they were added. Returns an empty list if there are no
-         * input key material values.
+         * Returns an unmodifiable {@code List} of input keying material values
+         * in the order they were added. Returns an empty list if there are no
+         * input keying material values.
```

Comment:
Sean asked about where do the byte array IKMs go. Here you should mention that an IKM added by `{@link #addIKM(byte[])}` is converted to a `SecretKeySpec` object. Same for `salts`. Same as in `ExtractThenExpand`.

Reply:
I see them. Plural noun does not match singular one at:
```
Input keying material values added by {@link Builder#addIKM(byte[])}
         * are converted to a {@code SecretKeySpec} object.
         * 
```
Maybe "{@code SecretKeySpec} objects"?

Reply:
Are you sure? When one call `addIKM(byte[])` multiple times, I can see in the code that each time a new `SecretKeySpec` object is added into the `ikms` list.